.github/workflows/bump_and_publish.yml

```diff
@@ -10,6 +10,14 @@ jobs:
   bump-and-publish:
     runs-on: ubuntu-latest
     steps:
+      - name: Check branch
```

Comment:
not necessarily needed as workflow will fail if we not triggered `main` or `crumbs-` branches anyways but this is an earlier exit